src/py-opentimelineio/opentimelineio-bindings/otio_serializableObjects.cpp

```diff
@@ -362,21 +363,26 @@ static void define_items_and_compositions(py::module m) {
              "in_offset"_a = RationalTime(),
              "out_offset"_a = RationalTime(),
              py::arg_v("metadata"_a = py::none()))
-        .def_property("transition_type", &Transition::transition_type, &Transition::set_transition_type)
-        .def_property("in_offset", &Transition::in_offset, &Transition::set_in_offset)
-        .def_property("out_offset", &Transition::out_offset, &Transition::set_out_offset)
+        .def_property("transition_type", &Transition::transition_type, &Transition::set_transition_type, "Kind of transition, as defined by the :class:`Type` enum.")
+        .def_property("in_offset", &Transition::in_offset, &Transition::set_in_offset, "Amount of the previous clip this transition overlaps, exclusive.")
+        .def_property("out_offset", &Transition::out_offset, &Transition::set_out_offset, "Amount of the next clip this transition overlaps, exclusive.")
         .def("duration", [](Transition* t) {
             return t->duration(ErrorStatusHandler());
             })
         .def("range_in_parent", [](Transition* t) {
             return t->range_in_parent(ErrorStatusHandler());
-            })
+            }, "Find and return the range of this item in the parent.")
         .def("trimmed_range_in_parent", [](Transition* t) {
             return t->trimmed_range_in_parent(ErrorStatusHandler());
-            });
+            }, "Find and return the timmed range of this item in the parent.");
+
+    py::class_<Transition::Type>(transition_class, "Type", R"docstring(
+Enum encoding types of transitions.
 
+This is for representing “Dissolves” and “Wipes” defined by the multi-source effect as defined in section 7.6.3.2 of `SMPTE 258M-2004 <https://ieeexplore.ieee.org/servlet/opac?punumber=7291837>`_.
```

Comment:
That document has restricted, and paywalled access. Do we want to point to it? I argue no. Also the enumeration in OTIO has only dissolve. I don't think we want to formalize the SMPTE set of wipes without discussion. The majority of them are exotic analog effects from the mid to late twentieth century, such as "wiggly barndoor zigzag", "spiralling inverted bananas (counter clockwise)", and "I'm an intern at a public broadcasting station and no one is looking hahahaha". If we do adopt an enumeration of more than one element, we should discuss at the TSC.

Reply:
I removed the sentence.